src/neptune/exceptions.py

```diff
@@ -1223,3 +1224,9 @@ def __init__(self, disk_utilization: float, utilization_limit: float):
         super().__init__(
             message.format(disk_utilization=disk_utilization, utilization_limit=utilization_limit, **STYLES)
         )
+
+
+class NeptuneInvalidQueryException(NeptuneException):
+    def __init__(self, nql_query: str):
+        message = f"""provided nql query: {nql_query} is invalid"""
```

Comment:
Could be lengthy, so perhaps this order is better:

```suggestion
        message = f"""The provided NQL query is invalid: {nql_query}"""
```

This would probably be a good place to include a link to the docs, in case people need help with the syntax. The URL will be:

```
https://docs.neptune.ai/usage/nql/
```

Reply:
How abut this

```
class NeptuneInvalidQueryException(NeptuneException):
    def __init__(self, nql_query: str):
        message = f"""
The provided NQL query is invalid: {nql_query}.
Learn more: https://docs.neptune.ai/usage/nql/
"""
        super().__init__(message)

```